styles.js

```diff
@@ -0,0 +1,37 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const ham = document.querySelector('.hamburger');
+  const menu = document.querySelector('.nav-menu');
+  const close = document.querySelector('.fa-solid.fa-xmark');
+  const bars = document.querySelector('.fa-solid.fa-bars');
+
+  ham.addEventListener('click', () => {
+    menu.classList.toggle('show');
+    if (menu.classList.contains('show')) {
+      bars.style.display = 'none';
+      close.style.display = 'block';
+    } else {
+      bars.style.display = 'block';
+      close.style.display = 'none';
+    }
+  });
+
+  close.addEventListener('click', () => {
+    menu.classList.remove('show');
+    bars.style.display = 'block';
+    close.style.display = 'none';
+  });
```

Comment:
- [ ] According to the project requirement, 👇🏻  When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. Kindly address this 😉 

![image](https://user-images.githubusercontent.com/79058364/217652145-79a0b2e1-9a84-4bad-aa5d-86b950e8f208.png)